packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -117,10 +117,16 @@
 
 .mdc-chip__icon--leading {
   transition: opacity $mdc-chip-opacity-animation-duration linear;
+
+  // Speed up delay to bridge gap between leading icon and checkmark transition.
+  transition-delay: -50ms;
```

Comment:
Why is this negative?

Reply:
By delaying the transition of the checkmark it also delayed the appearance of the leading icon, and so I used the transition-delay feature to actually speed it up to bridge the gap from checkmark fading out and leading icon fading in. Helps the transition feel/look more natural.

Reply:
Ah my bad! I thought this was the `transition-duration`. 

Since these numbers are important, I'd move them to the `_variables.scss` file.

Once that's in, I'll LGTM it.

Reply:
Done!